protocol/x/revshare/keeper/revshare.go

```diff
@@ -175,6 +175,11 @@ func (k Keeper) GetAllRevShares(
 	makerFees := fill.MakerFeeQuoteQuantums
 	netFees := big.NewInt(0).Add(takerFees, makerFees)
 
+	// net fees is 0 in case of liquidations where maker and taker fees are 0
```

Comment:
This comment is not general enough. Net fees can be zero if `takerFee > 0` and `makerFee < 0`. It should just say "when net fee is zero, not rev share is generated from the fill'